lib/matplotlib/tests/test_mathtext.py

```diff
@@ -114,6 +114,7 @@
 # default) and only png outputs, in order to minimize the size of baseline
 # images.
 lightweight_math_tests = [
+    r'$\sqrt[ab]{123}$',  # github issue #8665
```

Comment:
What's up with the `ab` alignment? It doesn't appear to follow the baseline to me, but align to top instead.

Reply:
This only appears with text.hinting = "none" (which is used by tests, but is not the default otherwise).  I suspect this is just one of the vertical alignments problems mentioned in https://github.com/matplotlib/matplotlib/pull/5414.
Also manually testing with a large text size (e.g. 48) allows verifying that there is no top-alignment.